install.md

```diff
@@ -159,7 +159,13 @@ brew install gpgme
 sudo zypper install libgpgme-devel device-mapper-devel libbtrfs-devel glib2-devel
 ```
 
-Make sure to clone this repository in your `GOPATH` - otherwise compilation fails.
+For newer versions of Go with modules support, after installing the above prereqs, you can build and install Skopeo using a simple `go get`:
```

Comment:
Could you define newer?  IE >= golang 1.14, 1.15, 1.16?

Reply:
It's not version-specific; it depends on environment variable content as well as the version of Go.  I'll include a URL.